evaluator/evaluator.go

```diff
@@ -25,6 +25,10 @@ func NewEvaluator(templates map[string]*hclast.File, schema []*schema.Template,
 		Type:  hilast.TypeString,
 		Value: c.TerraformEnv,
 	}
+	varMap["terraform.workspace"] = hilast.Variable{
```

Comment:
Please add a test case for `${terraform.workspace}` syntax :)
Example:
https://github.com/wata727/tflint/blob/3cf4a7eff3d7869da768ae3e51d8699d77c4c1df/evaluator/evaluator_test.go#L267-L295

Reply:
Yep sure, I'll add that in soon 😬

Reply:
Ready for review :+1: